src/ResourceManager/Insights/Commands.Insights/Metrics/GetAzureRmMetricDefinitionCommand.cs

```diff
@@ -14,20 +14,16 @@
 
 using System.Linq;
 using System.Management.Automation;
-using System.Text;
-using System.Threading;
 using Microsoft.Azure.Commands.Insights.OutputClasses;
 using Microsoft.Azure.Management.Monitor;
-using Microsoft.Azure.Management.Monitor.Models;
-using Microsoft.Rest.Azure.OData;
 
 namespace Microsoft.Azure.Commands.Insights.Metrics
 {
     /// <summary>
     /// Get the list of metric definitions for a resource.
     /// </summary>
-    [Cmdlet(VerbsCommon.Get, "AzureRmMetricDefinition"), OutputType(typeof(PSMetricDefinition))]
-    public class GetAzureRmMetricDefinitionCommand : MonitorClientCmdletBase
+    [Cmdlet(VerbsCommon.Get, "AzureRmMetricDefinition"), OutputType(typeof(PSMetricDefinition[]))]
```

Comment:
@pkamath13 @gucalder same comment about changing this output type back to the concrete type